sys/linux/auto.txt

```diff
@@ -3772,10 +4557,30 @@ ioam6_genl_policy_ns_sc$auto_ioam6 [
 	IOAM6_ATTR_SC_NONE	nlattr[IOAM6_ATTR_SC_NONE, void]
 ] [varlen]
 
+iocb$auto_record {
+	aio_data	intptr
+	aio_key		int32
+	aio_rw_flags	int32
+	aio_lio_opcode	int16
+	aio_reqprio	int16
+	aio_fildes	int32
+	aio_buf		intptr
+	aio_nbytes	intptr
+	aio_offset	intptr
+	aio_reserved2	intptr
+	aio_flags	int32
+	aio_resfd	int32
+}
+
+iovec$auto_record {
+	iov_base	ptr[inout, void]
```

Comment:
Input void does not make sense.
I think input `void*` should generally be `array[int8]`.

Reply:
How about system call parameters? Should I keep them as `void*`? Or maybe `intptr`?

Reply:
The logic should be the same everywhere: `void*` -> `ptr[inout, array[int8]]`.

Reply:
Done!